frontend/src/scenes/session-recordings/player/inspector/components/ItemComment.tsx

```diff
@@ -0,0 +1,47 @@
+import { useActions } from 'kea'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { notebookPanelLogic } from 'scenes/notebooks/NotebookPanel/notebookPanelLogic'
+import { InspectorListItemComment } from 'scenes/session-recordings/player/inspector/playerInspectorLogic'
+
+export interface ItemCommentProps {
+    item: InspectorListItemComment
+    expanded: boolean
+    setExpanded: (expanded: boolean) => void
+}
+
+export function ItemComment({ item, expanded, setExpanded }: ItemCommentProps): JSX.Element {
+    const { selectNotebook } = useActions(notebookPanelLogic)
+
+    return (
+        <div data-attr="item-comment">
+            <LemonButton noPadding onClick={() => setExpanded(!expanded)} fullWidth className="font-normal">
+                {expanded ? (
+                    <div className="p-2 text-xs border-t w-full flex justify-end">
+                        <LemonButton
```

Comment:
Does this button **have** to be  inside the other button? It's technically not valid html to do that, and I've seen it case problems before. (And I know we're not using SSR, but react 19 would also complain about this during hydration)

Reply:
ah, totally valid... i guess that's affecting every type of `item` in the inspector

definitely something to resolve, i know Cory wants to change the interactions around these items as well, so that "expand" and "jump to time" are more clear